packages/engine/lib/execution/src/lib.rs

```diff
@@ -0,0 +1,18 @@
+//! # Workers, language runners and the hEngine Package System
```

Comment:
I don't think this is a good opening doc line to be honest. We want to describe what the _point_ of the crate is. 

(Don't just copy paste this because I'm not spending too much time thinking about phrasing) Something like
Handles what logic is applied, and when, during simulation as well as how that work is distributed and executed.

Reply:
I improved the crate level documentation, should be far better now. Also adjusted the docs on `TaskMessage` as it contained outdated information and weird formatting.